homeworks/olena.ambitska_elena-ambitska/5-logic/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Basic JS</title>
+        <link rel="stylesheet" href="../GamesProject/styles/styles.css">
+        <link rel="stylesheet" href="styles.css">
+
+        <style>
+            .field {
+                display: flex;
+                column-gap: 10px;
+                margin: 20px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1 class="calc__title">Calculator v. 1</h1>
+        <div class="field">
+            <label>
+                <input id="firstValue" type="number" min="0" step="1" class="calc__input">
+            </label>
+            <label>
+                <select name="operation" id="operation" class="calc__dropdown">
+                    <option value="+">+</option>
+                    <option value="-">-</option>
+                    <option value="/">/</option>
+                    <option value="*">*</option>
+                    <option value="**">**</option>
+                </select>
+            </label>
+            <label>
+                <input id="secondValue" type="number" min="0" step="1" class="calc__input">
+            </label>
+            <button id="calculate" type="button" class="calc__button">=</button>
+        </div>
+        <section class="counter__section">
```

Comment:
Hey, @elena-ambitska, so good so far. Some comments from me:
1. For `*` task, the calculator page should be integrated into your project.
2. Please align pictures with inputs.
3. Place  the resulting cards in the column of 10 pcs

Reply:
@vladyslav-yermolin-moc done for every thread

<details>

<img width="1433" alt="Screenshot 2023-11-04 at 15 43 33" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/74259586/59ec17fa-3c02-498f-9998-320593c41ffd">

</details>

Reply:
Please align the columns 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/3dd6476e-4f93-44fa-80af-1fdd7cd4850f)

Reply:
Please move your files to the project folder 
<img width="392" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/bcdb0a7c-31b1-4553-9550-2a2350c354f0">

Reply:
Please update the header
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/81ab8194-f5a9-43b3-8fa3-3f88d69bca38)

Reply:
@vladyslav-yermolin-moc done 

<img width="1433" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/74259586/f5ddf634-2a57-4f8b-a0e5-6ac8d572ff9e">